src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/Common/DbConnectionStringCommon.cs

```diff
@@ -1124,7 +1124,7 @@ internal static class DbConnectionStringSynonyms
         internal const string APP = "app";
 
         //internal const string ApplicationIntent    = APPLICATIONINTENT;
-        internal const string APPLICATIONINTENT = "ApplicationIntent";
+        internal const string APPLICATIONINTENT = "applicationintent";
```

Comment:
This class does not require lowercase changes, but changes done for consistency.

Reply:
Where is the test for the reported issue?

Reply:
Ref: https://github.com/dotnet/SqlClient/pull/585#discussion_r433436374